web-src/src/components/PromptSessionSideView.js

```diff
@@ -116,7 +118,7 @@ export function PromptSessionSideView({ isOpenPromptEditor, onTogglePrompt, ...p
           <SavePromptButton />
           <ResetButton />
         </Flex>
-        <GenerateButton />
+        <GenerateButton disabled={disableTextGeneration} />
```

Comment:
Should the `SavePromptButton` above be disabled as well? 🤔 Otherwise, clients might store and share broken prompts with their organization.

Reply:
That's a good point.